packages/legend-application-studio/src/components/workspace-setup/WorkspaceSetup.tsx

```diff
@@ -218,131 +296,172 @@ export const WorkspaceSetup = withWorkspaceSetupStore(
+                          </div>
+                          <CustomSelectorInput
+                            className="workspace-setup__selector__content__input"
+                            options={workspaceOptions}
+                            disabled={
+                              !setupStore.currentProject ||
+                              !setupStore.currentProjectConfigurationStatus ||
+                              !setupStore.currentProjectConfigurationStatus
+                                .isConfigured ||
+                              setupStore.loadProjectsState.isInProgress ||
+                              setupStore.loadWorkspacesState.isInProgress
+                            }
+                            isLoading={
+                              setupStore.loadWorkspacesState.isInProgress
+                            }
+                            onChange={onWorkspaceChange}
+                            formatOptionLabel={formatWorkspaceOptionLabel}
+                            value={selectedWorkspaceOption}
+                            placeholder={
+                              setupStore.loadWorkspacesState.isInProgress
+                                ? 'Loading workspaces...'
+                                : !setupStore.currentProject
+                                ? 'In order to choose a workspace, a project must be chosen'
+                                : workspaceOptions.length
+                                ? 'Choose an existing workspace'
+                                : 'You have no workspaces. Please create one to proceed...'
+                            }
+                            isClearable={true}
+                            escapeClearsValue={true}
+                            darkMode={true}
+                          />
+                        </div>
+                      </div>
+                    </form>
                   </div>
-                  <div className="workspace-setup__actions">
-                    <button
-                      className="workspace-setup__next-btn btn--dark"
-                      onClick={handleProceed}
-                      disabled={
-                        !setupStore.currentProject ||
-                        !setupStore.currentProjectConfigurationStatus ||
-                        !setupStore.currentProjectConfigurationStatus
-                          .isConfigured ||
-                        !setupStore.currentWorkspace ||
-                        setupStore.createWorkspaceState.isInProgress ||
-                        setupStore.createOrImportProjectState.isInProgress
-                      }
-                    >
-                      Edit Workspace
-                    </button>
+                  <div className="workspace-setup__actions-combo">
+                    <div className="workspace-setup__actions">
+                      <div className="workspace-setup__actions__button">
+                        <form
```

Comment:
is this <form> needed, we had one <form> component before right?

Reply:
`<form>` includes all three parts (dropdown for projects, dropdown for workspace,  editWorkspace[go] button) in the previous code.

I changed the structure a little bit, as the picture below shows. 
The first `<form>` ended after the dropdown for workspace
<img width="710" alt="Screenshot 2023-12-06 at 11 57 40 AM" src="https://github.com/finos/legend-studio/assets/73408381/2243e9e8-bd84-45e2-a5f2-8637ec835993">

Reply:
I realized that using two forms is actually not helping trigger `go` after hitting `enter` unless the go button and selectors are wrapped within the same form.
I achieved the same goal by removing the form since the current css design is based on grouping go button with another two buttons.